redbot/cogs/mutes/mutes.py

```diff
@@ -251,104 +257,124 @@ async def _handle_channel_unmutes(self):
             guild = self.bot.get_guild(guild_id)
             for user, channels in users.items():
                 if len(channels) > 1:
+                    task_name = f"server-unmute-channels-{user}"
```

Comment:
This name needs to contain guild id so that tasks for the same user can be spawned in 2 different servers.